article/sections/_2_graph_importance.tex

```diff
@@ -0,0 +1,189 @@
+\item We then use the choice model to simulate outcome choices based on data simulated for Figure \ref{fig:DA_causal_2} through Figure \ref{fig:BIKE_causal_2}.
+\item We apply the do-operator, as above, to reduce the travel distance for all individuals in the dataset and to emulate a company's decision to move its employees closer to campus.
+\item We use the estimated relationships for this causal graph to simulate all the explanatory variables in Figure \ref{fig:DA_causal_2} through Figure \ref{fig:BIKE_causal_2}(namely the variables affected by travel distance).
+\item We use the estimated choice model to produce outcomes based on the simulated data from the previous step.
+
+\end{enumerate}
+
+We repeat this simulation process several times and recover the choice probability of all modes for all individuals.
+For our purposes, we focus on car-centric modes (Drive alone, Shared ride with another person, 
+and a shared ride with two or more individuals).
+We compute the difference in mode choice probabilities from different models based on the two constructed DAGs.
+
+
+
+We then plot histograms of the computed differences between the average probability of an individual
+in our sample choosing a car centric mode before and after implementing a policy or intervention
+aimed at reducing travel distance.
+These differences are plotted under the two different assumptions about the data generating process illustrated in the causal graphs above.
+Figure \ref{fig:histogram_probability} highlights the bias between the estimated probability of an average individual choosing a car centric mode.
+This difference shows the importance of considering the data generating process when estimating transportation demand models aiming to forecast the impact of proposed policies.
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{histogram_selection_on_obs.png}
+   \caption{Histograms of the probability of choosing Car Centric Modes under Different Data Generating processes.}
+   \label{fig:histogram_probability}
+\end{figure}
+
+The data generating process might not be easily distinguishable in the majority of situations, mainly due to the complexity of the real world.
+Therefore, constructing a causal graph that represents the data generating process as much as possible is not an easy task.
+To prepare readers to create causal graphs, the next section will provide a brief overview of them and their history in choice modelling.
+Then, Section \ref{sec:graph-construction} explores this topic and includes some guidance on how to build causal graphs representing the researchers beliefs about the data generating process.
+Section \ref{sec:graph-testing} follows up with guidance on how to test one's causal graphs against one's data.
```

Comment:
```suggestion
Note that in the simulation presented above, we assumed that the true data generating process is known to the analyst. 
Our goal in doing that is to illustrate how the bias from misrepresenting the data generating process can be arbitrarily large, especially when the analyst takes a naive approach of only focusing on the variables directly impacted by the policy at hand. 
The next section will provide a brief overview of causal graphs and their history in choice modelling.
Then, Section \ref{sec:graph-construction} will provide guidance on how to build causal graphs representing the researchers beliefs about the data generating process.
Section \ref{sec:graph-testing} follows up with guidance on how to test one's causal graphs against one's data.
```

Reply:
I like your proposed change @hassan-obeid, but I still think that we should re-emphasize how our example is not reflective of what's going on in the real world. I know we mention this in line 40 above, but I think it's worth it to repeat in here. I also thing that the transition from talking about the simulation to talking about the next sections talk about is a little rough. Let's talk today.

Reply:
See #163